MODULE.md

```diff
@@ -168,6 +160,8 @@ If the PoA admin decides they no longer wish for a validator to be signing block
 
 The Token Factory module as it is implemented on the Manifest Network, allows any user to have granular control over the creation and management of tokens on the Manifest Network. The creator can mint, burn, edit, and transfer tokens to other accounts from any account.
 
+>_note:_ Native tokens are not supported by the Token Factory module. The module is designed to work with tokens created by the module itself.
```

Comment:
nit: SudoMint allows for native tokens to be minted.

also, tokenfactory tokens are native, just with a specific format for permissioned reasons

Reply:
[31aa9fc](https://github.com/liftedinit/manifest-ledger/pull/70/commits/31aa9fce40484e77734b819334eff9c72a04eb66)